### PR TITLE
Added hints setting to Module.

### PR DESCRIPTION
**Pull request by richese**
This setting gives acces to all features of i3bar protocol that are not handled by most modules but may be useful sometimes.

Rigth now this PR will add `"markup": "none"` to all outputs by default. This should prevent #181 from  happening but technically it should be the default anyway so I am not sure if to keep it there.

### CODE REVIEW

**Comment by enkore** [comment]
Looks good, but the terminology isn't right. The output of a module in i3pystatus is the same as an i3bar block, the key/value pairs of the block are not blocks in their own regard but just key/value pairs. It's a bit misleading in the i3bar docs that section 2.2 is titled "Blocks in detail", what they mean is the structure of blocks in detail, i.e. permitted keys and how the values of those keys should look like.

**Comment by richese** [comment]
I see what you mean. I'll rewrite & move the docs.

File: i3pystatus/core/modules.py [hunk 1]
```diff
@@ -13,6 +13,7 @@ class Module(SettingsBase):
         ('on_rightclick', "Callback called on right click (see :ref:`callbacks`)"),
         ('on_upscroll', "Callback called on scrolling up (see :ref:`callbacks`)"),
         ('on_downscroll', "Callback called on scrolling down (see :ref:`callbacks`)"),
+        ('hints', "Additional output blocks for module output (see :ref:`hints`)"),
```
Comment (richese) [inline i3pystatus/core/modules.py]: These refferences are not displaying correctly in the [documentation](http://docs.enkore.de/i3pystatus/i3pystatus.core.html#i3pystatus.core.modules.IntervalModule.settings).

Comment (enkore) [inline i3pystatus/core/modules.py]: Yeah sphinx doesn't parse normal strings in the code. It does show correctly in the module reference, though.

**Pull request merged**